docs/v0.6/en/user-guide/operations/configuration.md

```diff
@@ -193,15 +212,15 @@ All of these protocols except HTTP and gRPC are optional, the default values are
 |            | addr         | String  | Server address, "127.0.0.1:4003" by default                                     |
 |            | runtime_size | Integer | The number of server worker threads, 2 by default                               |
 
-#### Node options
+<!-- #### Node options
 
 There are also some node options in common:
 
 | Option | Key  | Type   | Description                                                |
 | ------ | ---- | ------ | ---------------------------------------------------------- |
-|        | mode | String | Node running mode, includes "standalone" and "distributed" |
+|        | mode | String | Node running mode, includes "standalone" and "distributed" | -->
```

Comment:
Keep it before we remove it.

Reply:
Are there any default values here?
Default values make users do not need to care about the options.